tests/vmi_cloudinit_test.go

```diff
@@ -87,19 +87,19 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			defer expecter.Close()
 
 			By("Checking that the VirtualMachineInstance serial console output equals to expected one")
-			resp, err := expecter.ExpectBatch(commands, timeout)
+			resp, err := tests.ExpectBatchWithValidatedSend(expecter, commands, timeout)
```

Comment:
Same here as in the login. I don't think that `tests.ExpectBatchWithValidatedSend` is suitable in this case since the expected expression is not a result of the sent command.